atlp-rwanda/octopus-bn-frontend
# #170947382 Users should be able to Login via Facebook and Google.

#### What does this PR do?
- Users should be able to Login via Facebook and Google. 

#### Description of Task to be completed?
- add click to social buttons
- handle click on social login
- create callback component

#### How should this be manually tested?
- npm close this repo `git clonehttps://github.com/Stackup-Rwanda/octopus-bn-frontend`
- check out to this branch `ft-social-login-170947382`
- npm install all required dependencies  `npm install`
- start client-server `npm run dev`
- if your testing this locally please create a .env file in the parent directory 
and those two variables
```
FACEBOOK_SERVER_CALL=http://localhost:3000/api/v1/auth/facebook
GOOGLE_SERVER_CALL=http://localhost:3000/api/v1/auth/google

```
Note: Make sure that your backend server is on

#### Screenshots (if appropriate)

![screencapture-accounts-google-signin-oauth-oauthchooseaccount-2020-04-28-12_21_49](https://user-images.githubusercontent.com/52036624/80476702-0654e180-894b-11ea-9640-159af5cd9bfe.png)


[Finishes #170947382]"
